### DISCUSSION TITLE
Instance ID token or FIS auth token for A/B testing device without using firebase_messaging

### DISCUSSION THREAD

**Discussion by fooshyangsong** [firebase/flutterfire#4938] [Q&A]
Hello everyone, I am currently working on A/B testing for my app. I required to get instance id token in order to validate my experiment on a test device. I had look through the helpline but it doesn't provide a sample for flutter.  https://firebase.google.com/docs/ab-testing/abtest-config?authuser=0#validate_your_experiment_on_a_test_device

**Comment by digosantos** [comment]
Did you manage to solve that?
